clinica/pydra/pet_volume/pipeline.py

```diff
@@ -74,7 +74,7 @@ def build_core_workflow(name: str = "core", parameters: dict = {}) -> Workflow:
 
     from clinica.pydra.shared_workflows.smoothing import build_smoothing_workflow
     from clinica.pydra.utils import sanitize_fwhm
-    from clinica.utils.pet import get_suvr_mask
+    from clinica.pipelines.pet.utils import get_suvr_mask
```

Comment:
Could you move this line just before the import of `build_smoothing_workflow` ? This will make the code formatter happy and turn the CI green.